src/app/fdctl/configure/workspace.c

```diff
@@ -48,7 +48,14 @@ fdctl_obj_new( fd_topo_t const *     topo,
   } else if( FD_UNLIKELY( !strcmp( obj->name, "metrics" ) ) ) {
     fd_metrics_new( laddr, VAL("in_cnt"), VAL("out_cnt") );
   } else {
-    FD_LOG_ERR(( "unknown object `%s`", obj->name ));
+    /* if the object specified the properities necessary to be concrete */
+    ulong align = fd_pod_queryf_ulong( topo->props, ULONG_MAX, "obj.%lu.align", obj->id );
```

Comment:
??? This does not new the object. Who is going to call new?

Reply:
The user.

Reply:
it explicitly doesnt happen here. It is your responsibility to set the object up where you actually intend to use it.

Reply:
Added a better comment there.